pom.xml

```diff
@@ -15,16 +15,16 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.compiler.source>11</maven.compiler.source>
 		<maven.compiler.target>${maven.compiler.source}</maven.compiler.target>
-		<junit.jupiter.version>5.9.1</junit.jupiter.version>
+		<junit.jupiter.version>5.10.1</junit.jupiter.version>
 		<junit.platform.version>1.8.2</junit.platform.version>
 		<slf4j.version>1.7.32</slf4j.version>
 		<kafka.version>2.8.0</kafka.version>
-		<kafka-connect-api.version>2.7.1</kafka-connect-api.version>
+		<kafka-connect-api.version>2.7.2</kafka-connect-api.version>
```

Comment:
Since MSK Connect is running on Kafka Connect 2.7.1, I would prefer to keep this unchanged